.gitignore

```diff
@@ -299,3 +299,8 @@ sketch
 static
 
 # End of https://www.toptal.com/developers/gitignore/api/flask,react,macos,node
+
+# Elastic Beanstalk Files
```

Comment:
we want these config files to be ignored? I thought it would be helpful to version these.